<launch>    

    <node name="moveit_sensor_switcher" type="mux" pkg="topic_tools" args="moveit_sensor camera/depth/points camera_base/depth/points mux:=mux_cam_depth" />

    <include file="$(find pmb2_arm_moveit_config)/launch/moveit_planning_gazebo.launch" />

    <arg name="config" default="$(find pmb2_apps)/config/config_global_moveit_palbator.yaml" doc="Path to parameter config file"/>
    <arg name="grasp_config" default="$(find pmb2_grasp)/config/pmb2_grasp_data.yaml" doc="Path to grasp parameter config file"/>

    <node name="moveit_global_controller" pkg="pmb2_apps" type="moveit_global_controller.py" output="screen">
        <rosparam file="$(arg config)"/>
        <rosparam file="$(arg grasp_config)"/>
        <param name="grasp_group_name" value="pmb2_gripper" />
    </node>

</launch>